subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/AndroidPerformanceTestFixture.groovy

```diff
@@ -29,4 +33,25 @@ trait AndroidPerformanceTestFixture {
         runner.assumeShouldRun()
         AndroidTestProject.projectFor(runner.testProject)
     }
+
+    void configureProjectJavaHomeToJdk11() {
```

Comment:
I don't like changing the Java home for the performance test. Could we run the performance test directly on Java 11? We don't necessarily need to do this as part of this PR.

In general, I think it is time to switch to using Java 11 for performance tests.

Reply:
Ok, I will open an issue to fix that in a separate PR.

Reply:
https://github.com/gradle/gradle/issues/21646